airo-robots/airo_robots/manipulators/hardware/ur_rtde.py

```diff
@@ -1,20 +1,144 @@
 import time
+from multiprocessing import Array, Process, Value
 from typing import Optional
 
 import numpy as np
 from airo_robots.awaitable_action import AwaitableAction
 from airo_robots.grippers import ParallelPositionGripper
 from airo_robots.manipulators.position_manipulator import ManipulatorSpecs, PositionManipulator
 from airo_spatial_algebra import SE3Container
-from airo_typing import HomogeneousMatrixType, JointConfigurationType
-from loguru import logger
+from airo_typing import HomogeneousMatrixType, JointConfigurationType, WrenchType
 from rtde_control import RTDEControlInterface
 from rtde_receive import RTDEReceiveInterface
 
 RotVecPoseType = np.ndarray
 """ a 6D pose [tx,ty,tz,rotvecx,rotvecy,rotvecz]"""
 
 
+def _torque_worker(
```

Comment:
Can you add type annotations here? See `airo-typing` for some relevant types.

Can you also add a docstring to explain what this function is for, and to describe the arguments and return type?